README.md

```diff
@@ -336,6 +340,28 @@ avalanche-network-runner control create-subnets '[{}, {"participants": ["node1",
 
 ```
 
+To get a list of all the subnet ids:
+
+```bash
+curl -X POST -k http://localhost:8081/v1/control/listsubnets
+
+# or
+avalanche-network-runner control list-subnets
+```
+
+_Note: To create a blockchain, the vm binary for it should be present under `AVALANCHEGO_PLUGIN_DIR`, with a filename equal to the vm id.
+But, ANR receives a vm name as param, and uses it to generate a vm id to look for. So there is a need to get the expected
```

Comment:
I'm not sure what the rest of this paragraph means.

Reply:
ok simplified the parragraph